### DISCUSSION TITLE
Special variables for `show-text`

### DISCUSSION THREAD

**Discussion by lamyergeier** [mpv-player/mpv#14109] [Q&A]
In input.conf

```
i show-text "Filename: ${filename}"
```

This shows the filename when i is pressed.

Where can I find a list of all of these special variables? For example I want to `show-text` the `dirname` (directory path)  of the video.

**Comment by guidocella** [comment]
https://mpv.io/manual/master/#property-list

But there is only `${path}` and no directory property.
